sbt-plugin-test/testFramework/src/main/scala/sbttest/framework/DummyTask.scala

```diff
@@ -34,7 +36,8 @@ final class DummyTask(
 
   def execute(eventHandler: EventHandler, loggers: Array[Logger],
       continuation: Array[Task] => Unit): Unit = {
-    continuation(execute(eventHandler, loggers))
+    val tasks = execute(eventHandler, loggers)
+    runner.taskBlock.foreach(_ => continuation(tasks))
```

Comment:
Will other testing frameworks have to do that, now? Or is it specific to the Dummy framework?